lib/portage/dbapi/bintree.py

```diff
@@ -1234,11 +1234,35 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", "true")
+        if portage_trust_helper == "true":
+            return 0
```

Comment:
```suggestion
        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", None)
        if portage_trust_helper is None:
            return
```

Reply:
This would allow the user to disable the setting by setting it to empty string.

Reply:
Done, good idea.